seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/flow/SourceFlowLifeCycle.java

```diff
@@ -177,7 +177,7 @@ public void requestSplit() {
                             enumeratorTaskAddress)
                     .get();
         } catch (InterruptedException | ExecutionException e) {
-            log.warn("source request split failed [{}]", e);
+            log.warn("source request split failed.", e);
```

Comment:
Why change it?

Reply:
> Why change it?

Exceptions will not be spliced ​​into {}, this usage is wrong,